### PR TITLE
R4R: chore: update changelog

### PR DESCRIPTION
**Pull request by okwme**

### CODE REVIEW

**Review by yaruwangway — APPROVED** [review]

**Comment by okwme** [comment]
R4R @yaruwangway

File: CHANGELOG.md [hunk 1]
```diff
@@ -29,29 +29,43 @@ Types of changes (Stanzas):
...
 "State Machine Breaking" for breaking the AppState
 
+
 Ref: https://keepachangelog.com/en/1.0.0/
 -->
 
 # Changelog
 
 ## [Unreleased]
+* (gaia) Bump [Liquidity](https://github.com/gravity-devs/liquidity) module to [v1.4.4](https://github.com/Gravity-Devs/liquidity/releases/tag/v1.4.4).
```
Comment (yaruwangway) [inline CHANGELOG.md]: liquidity uses v1.4.2, so we should delete line 39.

Comment (yaruwangway) [reply CHANGELOG.md]: liquidity uses v1.4.2 or 1.4.5 if liquidity module is going to release

Comment (dongsam) [reply CHANGELOG.md]: Hi @okwme @yaruwangway,  I'm Sorry for releasing liquidity versions [v1.4.4](https://github.com/Gravity-Devs/liquidity/releases/tag/v1.4.4) and [v1.4.5](https://github.com/Gravity-Devs/liquidity/releases/tag/v1.4.5) that failed to comply with the semantic versioning, I changed v1.4.4 and v1.4.5 to unusable and released [v1.4.6](https://github.com/Gravity-Devs/liquidity/releases/tag/v1.4.6), which containing only bump the sdk version to v0.44.6. 

Should I open a PR to bump liquidity to v1.4.6 on the main branch? Or if there's anything else I can help you with, please let me know.

Comment (okwme) [reply CHANGELOG.md]: thanks @dongsam !
please let us know when the version of liquidity is ready with cosmos sdk v0.45.1!

Comment (okwme) [reply CHANGELOG.md]: ah I see it is already released as well, thank you!

Comment (yaruwangway) [reply CHANGELOG.md]: Hi @dongsam , thank you ! we will later add more tests in gaia repo for testing compatibility of versions

Comment (yaruwangway) [inline CHANGELOG.md]:
```suggestion
* (gaia) Bump [Liquidity](https://github.com/gravity-devs/liquidity) module to [v1.4.6](https://github.com/Gravity-Devs/liquidity/releases/tag/v1.4.6).
```

Comment (okwme) [reply CHANGELOG.md]: we haven't done this yet tho. maybe fine to leave and then let's merge in a PR for using 1.4.6 quickly

File: CHANGELOG.md [hunk 2]
```diff
@@ -29,29 +29,43 @@ Types of changes (Stanzas):
...
 ## [Unreleased]
+* (gaia) Bump [Liquidity](https://github.com/gravity-devs/liquidity) module to [v1.4.6](https://github.com/Gravity-Devs/liquidity/releases/tag/v1.4.6).
+* (gaia) Bump [IBC](https://github.com/cosmos/ibc-go) module to [2.0.3](https://github.com/cosmos/ibc-go/releases/tag/v2.0.3).
+* (gaia) [#1230](https://github.com/cosmos/gaia/pull/1230) Fix: update gRPC Web Configuration in `contrib/testnets/test_platform`.
+* (gaia) [#1135](https://github.com/cosmos/gaia/pull/1135) Fix rocksdb build tag usage.
+* (gaia) [#1160](https://github.com/cosmos/gaia/pull/1160) Improvement: update state sync configs.
+* (gaia) [#1208](https://github.com/cosmos/gaia/pull/1208) Update statesync.bash.
+* * (gaia) Bump [Cosmos-SDK](https://github.com/cosmos/cosmos-sdk) to [v0.44.6](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.44.6)
+* (gaia) Bump [Versions](https://github.com/cosmos/gaia/pull/1100) of various smaller dependencies, remove the Cosmos SDK replace statement, update `initiClientCtx` params, ensure `stdout` and `stderr` are handled correctly in the CLI.
+* (gaia) Update links in docs [#1125](https://github.com/cosmos/gaia/issues/1125)
```
Comment (yaruwangway) [inline CHANGELOG.md]: does file change also goes to changelog ?

Comment (okwme) [reply CHANGELOG.md]: not typically
```suggestion
```

**Pull request merged**